client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -193,14 +199,17 @@ export const PrescriptionLineEditForm: React.FC<
       return;
 
     const quantity = inputUnitQuantity === undefined ? 0 : inputUnitQuantity;
-    setIssueUnitQuantity(quantity);
+    const qtyWithDoses = displayInDoses ? quantity / item?.doses : quantity;
```

Comment:
Think this should be called `unitQuantity` rather than `qtyWithDoses` - but if that is the case, input shouldn't be called `inputUnitQuantity`. I think better would be for this function to stay as is, and caller could do something like `handleIssueQuantityChange(dosesToUnitQuantity(value))`

Reply:
Probably related - why my doses per unit is 10 - the pluralisation is wrong, if I type 10, I can't backspace back to 1? But can highlight can change to 2, or 20, but then can't change 20 back to 2:

https://github.com/user-attachments/assets/7dab5d92-71f5-4b8e-84ea-c103c78887e4